src/Client/Settings.java

```diff
@@ -1071,18 +1066,12 @@ public static void definePresets(Properties props) {
     LOG_FORCE_LEVEL.put(
         "custom", getPropBoolean(props, "log_force_level", LOG_FORCE_LEVEL.get("default")));
 
-    Util.hasXdgOpen = Util.detectBinaryAvailable("xdg-open", "URL opening");
```

Comment:
Util.hasXdgOpen should still be set. It is used by LinkeOpener class, and it is kind of annoying that 
`[  INFO][21:23:54] xdg-open: /usr/bin/xdg-open`
is always output with
`[  INFO][21:23:54] Loaded settings`
every time that settings are applied.

Before it was caching the value of "hasXdgOpen" from startup.

Reply:
Great catch, thank you - will definitely fix this